sdk/healthinsights/azure-health-insights-radiologyinsights/src/main/java/com/azure/health/insights/radiologyinsights/RadiologyInsightsAsyncClient.java

```diff
@@ -272,160 +283,315 @@ public final class RadiologyInsightsAsyncClient {
      * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
      * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
      * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
-     * @return the {@link PollerFlux} for polling of provides status details for long running operations.
+     * @return the {@link PollerFlux} for polling of response for the Radiology Insights request.
+     */
+    @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
+    public PollerFlux<RadiologyInsightsData, RadiologyInsightsInferenceResult> beginInferRadiologyInsights(String id,
+        RadiologyInsightsData resource, List<String> expand) {
+        // Generated convenience method for beginInferRadiologyInsightsWithModel
+        RadiologyInsightsJob job = new RadiologyInsightsJob();
+        job.setJobData(resource);
+        RequestOptions requestOptions = new RequestOptions();
+        if (expand != null) {
+            for (String paramItemValue : expand) {
+                if (paramItemValue != null) {
+                    requestOptions.addQueryParam("expand", paramItemValue, false);
+                }
+            }
+        }
+        return serviceClient.beginInferRadiologyInsightsWithModelAsync(id, BinaryData.fromObject(job), requestOptions);
+    }
+
+    /**
+     * Create Radiology Insights job
+     *
+     * Creates a Radiology Insights job with the given request body.
+     *
+     * @param id The unique ID of the job.
+     * @param resource The resource instance.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the {@link PollerFlux} for polling of response for the Radiology Insights request.
      */
-    @Generated
     @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
-    public PollerFlux<RadiologyInsightsResult, RadiologyInsightsInferenceResult>
-        beginInferRadiologyInsights(RadiologyInsightsData radiologyInsightsData) {
+    public PollerFlux<RadiologyInsightsData, RadiologyInsightsInferenceResult> beginInferRadiologyInsights(String id,
+        RadiologyInsightsData resource) {
         // Generated convenience method for beginInferRadiologyInsightsWithModel
```

Comment:
same here, if it is not generated method, but customized one, better to remove the comment.
btw, could you please let me know what is the customization code change you have on this method?

Reply:
removed comments